doc/source/visualization.rst

```diff
@@ -163,7 +163,8 @@ For labeled, non-time series data, you may wish to produce a bar plot:
    plt.figure();
 
    @savefig bar_plot_ex.png
-   df.iloc[5].plot.bar(); plt.axhline(0, color='k')
+   df.iloc[5].plot.bar();
+   plt.axhline(0, color='k')
```

Comment:
I think we probably leave the semicolon at theend of the second line, to avoid displaying the plot representation. Did you render the html?

Reply:
Added semicolon. Build looks good.